### PR TITLE
update create sql file

### PR DESCRIPTION
**Pull request by haihuawu**
1. separate location table from note table
2. create note_location weak entity
3. create post weak entity
4. add state table

### CODE REVIEW

File: create.sql [hunk 1]
```diff
@@ -34,23 +34,44 @@ create table friendship(
...
+	latitude decimal(10, 8) not null, -- [-90, 90]*/
 	radius int not null,
 	visibility varchar(7) not null,
 	start_time timestamp not null,
 	end_time timestamp not null,
 	repetition varchar(5) not null, -- repeat is a keyword
 	allow_comments boolean not null,
-	unique (account_id, start_time, end_time, longitude, latitude, radius),
-	foreign key (account_id) references account(id),
+	/*unique (account_id, start_time, end_time, longitude, latitude, radius),*/
```
Comment (liyun-li) [inline create.sql]: Remove comments

File: create.sql [hunk 2]
```diff
@@ -14,12 +14,12 @@ create table account(
...
 );
 
-create table location(
+/*create table location(
 	account_id int primary key,
 	longitude decimal(11, 8) not null, -- [-180, 180]
 	latitude decimal(10, 8) not null, -- [-90, 90]
 	foreign key (account_id) references account(id)
-);
+);*/
```
Comment (liyun-li) [inline create.sql]: Remove

File: create.sql [hunk 3]
```diff
@@ -34,23 +34,44 @@ create table friendship(
 
 create table note(
 	id int primary key auto_increment,
-	account_id int not null,
 	message varchar(140) not null,
-	longitude decimal(11, 8) not null, -- [-180, 180]
-	latitude decimal(10, 8) not null, -- [-90, 90]
+	/*longitude decimal(11, 8) not null, -- [-180, 180]
```
Comment (liyun-li) [inline create.sql]: Remove

File: create.sql [hunk 4]
```diff
@@ -65,14 +86,23 @@ create table notetag(
...
 
 create table comment(
+	id int primary key auto_increment,
 	account_id int not null,
 	note_id int not null,
-	description varchar(4096) not null,
-	primary key (account_id, note_id),
+	description varchar(140) not null,
+	primart key (id),
+	/*primary key (account_id, note_id), can not be a primary key*/
```
Comment (liyun-li) [inline create.sql]: Remove

File: create.sql (hunk 2 above)
Comment (haihuawu) [reply create.sql]: ok

File: create.sql (hunk 3 above)
Comment (haihuawu) [reply create.sql]: ok

File: create.sql (hunk 4 above)
Comment (haihuawu) [reply create.sql]: ok

**Pull request merged**